Source/WebCore/css/CSSProperties.json

```diff
@@ -5371,7 +5370,9 @@
         },
         "-webkit-background-size": {
             "codegen-properties": {
-                "related-property": "background-size",
+                "longhands": [
+                    "background-size"
+                ],
                 "name-for-methods": "Size",
                 "fill-layer-property": true
```

Comment:
I think `name-for-methods` and `fill-layer-property` are not relevant for shorthands.